tests/components/aprs/test_device_tracker.py

```diff
@@ -0,0 +1,338 @@
+"""Test APRS device tracker."""
+import unittest
+from unittest.mock import Mock, patch
+
+import aprslib
+
+import homeassistant.components.aprs.device_tracker as device_tracker
+from homeassistant.const import EVENT_HOMEASSISTANT_START
+from homeassistant.exceptions import PlatformNotReady
+
+from tests.common import get_test_home_assistant
+
+DEFAULT_PORT = 14580
+
+TEST_CALLSIGN = 'testcall'
+TEST_COORDS_NULL_ISLAND = (0, 0)
+TEST_FILTER = 'testfilter'
+TEST_HOST = 'testhost'
+TEST_PASSWORD = 'testpass'
+
+
+class TestAprsDeviceTracker(unittest.TestCase):
```

Comment:
We have been moving away from class based tests. Instead, just make it all functions and it should work.